core/src/test/java/io/questdb/std/datetime/millitime/DateFormatCompilerTest.java

```diff
@@ -143,6 +143,27 @@ public void testFormatDay() throws Exception {
         assertFormat("03", "dd", "2014-04-03T00:00:00.000Z");
     }
 
```

Comment:
to_str and to_date should support the same format so it'd be good to test that parse method supports D w W letters .

Reply:
Isn't this already covered by the three tests below (testFormatDayOfYear/WeekOfYear/WeekOfMonth?

Reply:
Ah, I see what you were referring. I think i added tests for that in last commit.